STM32Cube/Tasks/can_handler.c

```diff
@@ -1,18 +1,24 @@
 #include "can_handler.h"
+#include "flight_phase.h"
 
 
-uint32_t LED_state = 0;
 xQueueHandle busQueue;
 
 void can_handle_rx(const can_msg_t *message, uint32_t timestamp){
-	if(get_message_type(message) == MSG_LEDS_ON)
+	uint16_t msgtype = get_message_type(message);
```

Comment:
can fix in a future branch but if this if/else chain is going into production it should be case/switch for :sparkles: performance optimization :sparkles: 
(prob not even affects this small amt of msgs but still good practice?